lib/features/mailbox/presentation/mailbox_view.dart

```diff
@@ -297,6 +300,49 @@ class MailboxView extends GetWidget<MailboxController> {
     ]);
   }
 
+  void _openBottomSheetSpamMenuAction(BuildContext context, PresentationMailbox mailbox) {
+    final _spamActionsSupported = [
+      controller.mailboxDashBoardController.enableSpamRepot
+              ? MailboxActions.disableSpamReport
```

Comment:
Each indent corresponds to 1 tab
```suggestion
final _spamActionsSupported = [
    controller.mailboxDashBoardController.enableSpamRepot
        ? MailboxActions.disableSpamReport
        : MailboxActions.enableSpamReport
]
```